### DISCUSSION TITLE
Ignore frequent entry updates

### DISCUSSION THREAD

**Discussion by hkcomori** [FreshRSS/FreshRSS#6311] [Ideas]
To detect updates of existing article, I have set mark as unread when it is overwritten.
However, it is annoying to have marked some articles as unread repeatedly for minor modifications (typos, omissions, etc.).
I only want to know about meaningful changes, so I prefer to ignore such trivial changes.

In my experience, many articles are often modified repeatedly with minor revisions over a short period of time, and meaningful changes are published only after a long enough time has elapsed.
Therefore, I believe that ignoring updates of existing articles for a period of time would solve my problem.

I tried the `if an identical title already exists in the top n newest articles` option, but it did not work for this purpose.

Is there any other way?

**Comment by Alkarex** [comment]
At the moment, there is no feature to automatise what you are asking. It would require a new core feature or a FreshRSS extension.
PR welcome

**Reply by hkcomori** [reply]
Thanks, I will send PR if good ones are made.
